src/test/performance/performanceTracker.node.ts

```diff
@@ -0,0 +1,60 @@
+import { IDisposable } from '@fluentui/react';
+import TelemetryReporter from '@vscode/extension-telemetry';
+import { extensions } from 'vscode';
+import { JVSC_EXTENSION_ID, AppinsightsKey, Telemetry } from '../../platform/common/constants';
+import { StopWatch } from '../../platform/common/utils/stopWatch';
+import { IS_CI_SERVER } from '../ciConstants.node';
+import { sleep } from '../core';
+
+export class PerformanceTracker implements IDisposable {
+    stopWatch = new StopWatch();
+    durations: Record<string, number> = {};
+    checkpointCount = 0;
+    telemetryReporter: TelemetryReporter | undefined;
+    telemetryEnabled = IS_CI_SERVER && !process.env.VSC_JUPYTER_WARMUP;
```

Comment:
Same thing here, IS_CI_SERVER should be a function call (it will always return true I think). Unless this is another constant that means the same thing.

The reason it's a function is because in web we can't check process.env, so it has to be set outside.

Reply:
looks like IS_CI_SERVER is a const in ciConstants.node.ts